routes/docs/index.js

```diff
@@ -1,15 +1,38 @@
 const i18n = require('../../lib/i18n')
+const cheerio = require('cheerio')
 
 module.exports = (req, res) => {
   const docsReadme = i18n.docs[req.language]['/docs/README']
 
+  let html
+
+  if (docsReadme.html) {
+    html = docsReadme.html
+  } else {
+    // Temporary fix to collect all section HTML
+    // can remove when docsReadme.html becomes available
+    // from electron-i18n bump.
+    html = docsReadme.sections.reduce((readmeHTML, { html: sectionHTML }) => {
+      return readmeHTML + sectionHTML
+    }, '')
+  }
+
+  const $ = cheerio.load(html)
+  const headingSelector = 'h2, h3, h4, h5, h6'
+  const sections = $(headingSelector)
+    .map((_, heading) => {
+      // grab all content between headings
+      let content = $(heading)
+        .nextUntil(headingSelector)
+        .map((_, p) => $.html(p))
+        .get()
+        .join()
+      return { html: content }
+    })
+    .get()
+    .slice(2, -1) // slice out unused headings
```

Comment:
I was thinking more about explaining what headings are being removed and why (are those sections that we don't need?).